code/game/objects/items/cards_ids.dm

```diff
@@ -833,7 +825,7 @@
 /obj/item/card/id/chip_programmer
```

Comment:
it would be nice if this one had it's own icon.

Reply:
i dont even know what that is

Reply:
gonna tackle that in the next batch, i have some ideas